client/src/components/BuildingDetails.js

```diff
@@ -32,7 +32,7 @@ export default function BuildingDetails(props) {
               <Trans>Address</Trans>:
             </Col>
             <Col lg={9} span={11}>
-              {details.address}
+              {details.address} <Trans>no.</Trans> {parseInt(details.post_code, 10)}
```

Comment:
I agree to trim leading zeros but we can't convert the street number into an `int`.
We have either one entry for multiple numbers (ex. 19-21) or building numbers with a letter (ex. 42B)

Reply:
removed parseInt